WorkloadManagementSystem/Service/WMSUtilities.py

```diff
@@ -70,6 +73,46 @@ def getCREAMPilotOutput( proxy, pilotRef, pilotStamp ):
   result['StdErr'] = error
   return result
 
+def getARCPilotOutput( proxy, pilotRef ):
+  tmp_dir = mkdtemp()
+  myce = pilotRef.split(":")[1].strip("/")
+  gridEnv = getGridEnv()
+  mySite = getCESiteMapping()['Value'][myce]
+  WorkDB = gConfig.getValue('Resources/Sites/LCG/' + mySite + '/CEs/' + myce + '/JobListFile')
```

Comment:
It's maybe being picky, but to avoid nasty surprises, it's better to use os.path.join for that kind of path operation